osu.Game/Overlays/Rankings/Tables/CountriesTable.cs

```diff
@@ -62,35 +61,20 @@ public CountriesTable(int page, IReadOnlyList<CountryStatistics> rankings)
             }
         };
 
-        private class CountryName : OsuHoverContainer
+        private class CountryName : LinkFlowContainer
         {
-            protected override IEnumerable<Drawable> EffectTargets => new[] { text };
-
             [Resolved(canBeNull: true)]
             private RankingsOverlay rankings { get; set; }
 
-            private readonly OsuSpriteText text;
-            private readonly Country country;
-
             public CountryName(Country country)
+                : base(t => t.Font = OsuFont.GetFont(size: 12))
             {
-                this.country = country;
-
-                AutoSizeAxes = Axes.Both;
-                Add(text = new OsuSpriteText
-                {
-                    Font = OsuFont.GetFont(size: 12),
-                    Text = country.FullName ?? string.Empty,
-                });
-            }
-
-            [BackgroundDependencyLoader]
-            private void load(OverlayColourProvider colourProvider)
-            {
-                IdleColour = colourProvider.Light2;
-                HoverColour = colourProvider.Content2;
+                AutoSizeAxes = Axes.X;
+                RelativeSizeAxes = Axes.Y;
+                TextAnchor = Anchor.CentreLeft;
 
-                Action = () => rankings?.ShowCountry(country);
+                if (country.FullName != null)
```

Comment:
```suggestion
                if (!string.IsNullOrEmpty(country.FullName))
```
More robust that way.